content-test/selectors/siteMetadataSelectors.test.js

```diff
@@ -82,10 +82,15 @@ describe("siteMetadataSelectors", () => {
 
       assert.ok(result.parsedUrl);
     });
-    it("should set label prop", () => {
+    it("should set label prop without eTLD set", () => {
       let result = selectSiteProperties(validSpotlightSite);
 
-      assert.ok(result.label);
+      assert.equal(result.label, "cnn.com");
```

Comment:
I know it's cosmetically the same thing but if I want to change `validSpotlightSite` to use mockData (like it should because APIs change and object names don't stay the same) I would also have to change your test.